deploy/crds/yugabyte_v1alpha1_ybcluster_cr.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: yugabyte.com/v1alpha1
+kind: YBCluster
+metadata:
+  name: example-ybcluster
```

Comment:
not sure if we want to call this example-ybcluster. YB universe is the construct we use from within YugaWare. Would be good to run it by @rkarthik007 and see how we want to name this?